test/test-helper.js

```diff
@@ -35,6 +35,9 @@ before(function(done) {
     global.$            = jQuery;
     global.Handlebars   = benv.require('../bower_components/handlebars/handlebars.min.js');
     global.Ember        = benv.require('../bower_components/ember/ember.debug.js', 'Ember');
+    // account for fact that Ember 2.7 and later don't think we have a DOM while we are testing
```

Comment:
i went deep into a rabbit whole in the ember internals trying to figure out why this but I'm not sure how much this matters since it a) only affects tests and b) what we are doing here is equivalent to what would be happening were Ember to do this for us